src/node.cc

```diff
@@ -4075,9 +4075,9 @@ void EmitBeforeExit(Environment* env) {
   HandleScope handle_scope(env->isolate());
   Context::Scope context_scope(env->context());
   Local<Object> process_object = env->process_object();
-  Local<String> exit_code = FIXED_ONE_BYTE_STRING(env->isolate(), "exitCode");
+  Local<String> exit_code = env->exit_code_string();
   Local<Value> args[] = {
-    FIXED_ONE_BYTE_STRING(env->isolate(), "beforeExit"),
+    env->before_exit_string(),
```

Comment:
As a pattern, we only store strings on the `Environment` if we expect to use them more than once per isolate, but in the case of `beforeExit` that’s not the typical case, right?

Reply:
It's not the case for "exit" or "exitCode" either ... so we should either just make them all use `FIXED_ONE_BYTE_STRING` here or put them all in `env` ... I'm good either way so long as it's consistent.

That said, one could *in theory* have `beforeExit` happen multiple times ... we've had that happen to us a few times ;-)

Reply:
In that case I’d prefer to follow the pattern and use `FIXED_ONE_BYTE_STRING` for those as well, yeah :)

Reply:
+1... I'll keep `exitCode` on `env` since that is at least used in both `beforeExit` and `exit`. But will make `beforeExit` and `exit` each use `FIXED_ONE_BYTE_STRING`